test-cases/upgrades/multi-dc-rolling-upgrade.yaml

```diff
@@ -0,0 +1,41 @@
+test_duration: 120
+# Workloads
+stress_before_upgrade: cassandra-stress write no-warmup cl=QUORUM n=10100200 -schema 'replication(factor=3) compression=LZ4Compressor compaction(strategy=LeveledCompactionStrategy)' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=1000 -pop seq=1..10100200 -log interval=5
+stress_during_entire_upgrade: cassandra-stress write no-warmup cl=QUORUM n=20100200 -schema 'keyspace=keyspace_entire_test replication(factor=3) compression=LZ4Compressor compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=100 -pop seq=1..20100200 -log interval=5
+stress_after_cluster_upgrade: cassandra-stress read no-warmup cl=QUORUM n=20100200 -schema 'keyspace=keyspace_entire_test replication(factor=3) compression=LZ4Compressor' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=1000 -pop seq=1..20100200 -log interval=5
```

Comment:
This need to have something that uses a NetworkTopologyStartegy something like: "'NetworkTopologyStrategy', 'us-eastscylla_node_east': '3', 'us-west-2scylla_node_west': '3'"

You can look at the manager sanity jobs that uses multi-dc.